mesh/mesh.hpp

```diff
@@ -1409,6 +1409,12 @@ class Mesh
    /// should not be deleted by the caller.
    static FiniteElement *GetTransformationFEforElementType(Element::Type);
 
+   /** @brief For the face of a boundary element with the orientation @a o (3D),
+       return the transformation of the boundary element integration point @ ip
+       to the face element. */
+   static IntegrationPoint TransformBdrElementFace(Geometry::Type geom, int o,
```

Comment:
I recommend to keep the `To` in the name like `TransformBdrElementToFace` or `TransformBdrElementToFaceElement`.

Reply:
Agreed. Done in [5cec99a](https://github.com/mfem/mfem/pull/3943/commits/5cec99aaf62e2f8273fd5de8953ee7a8d57ae648).